README.md

```diff
@@ -42,6 +42,10 @@ the following dependencies as well: `graphicsmagick` and `cairo`
     sudo apt-get install graphicsmagick cairo
 
 ### Install `beaker`
+Before installing Beaker, make sure you have a `node_modules` directory in your project folder:
+
+    mkdir node_modules
+
```

Comment:
Why is this needed? Whenever I bootstrap a new `beaker` project. I never manually create a `node_modules` directory. The `npm install beaker` will do that for you.